app/views/shared/_email_languages.html.erb

```diff
@@ -0,0 +1,32 @@
+<%#
+locals:
+* f: from validated_form_for
+* labelledby: id of element that radio boxes are labelled by
+* selection: the current language selection
+%>
+<div class="margin-bottom-4">
+  <ul class="usa-input-list">
+    <% I18n.available_locales.each do |locale| %>
+      <% item_id = "email-locale-#{locale}" %>
+      <% label_id = SecureRandom.hex %>
+      <li class="col-6">
+        <%= f.input_field(:email_language,
+                          type: :radio,
+                          value: locale,
+                          checked: selection ?
+                            selection.to_s == locale.to_s :
+                            (I18n.locale.to_s == locale.to_s),
+                          class: 'usa-radio__input',
+                          id: item_id,
+                          aria: { labelledby: "#{labelledby} #{label_id}" }) %>
```

Comment:
Did you consider to use the `fieldset` + `legend` approach from the [USWDS guidance](https://designsystem.digital.gov/components/form-controls/#radio-buttons) instead of the double label? One notable difference I find in how the screen readers treat this (at least for macOS VoiceOver) is as implemented here, the label "Please select your language preference below" is read for every radio button change, _before_ the option name (vs. after in USWDS guidance). I can imagine for someone navigating through the options, it would become quite tedious to hear this label each time.

Not sure if it might make a difference to reverse the order of the space-separated ID set here, though it seems simplest to use the `fieldset` + `legend`, which also benefits from being semantically treated as a [group](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_group_role) by [default](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#Technical_summary).

Reply:
`<fieldset>` + `<legend>` ended up having a serif font and didn't match the visual design, so I opted to use the `<label>` like this. I'll poke around and see if we have some stuff to make the fonts match

Reply:
Updated in dfe7078db46fd3f5087e3ec21e5025d1a5c267fe

Reply:
> `<fieldset>` + `<legend>` ended up having a serif font and didn't match the visual design, so I opted to use the `<label>` like this. I'll poke around and see if we have some stuff to make the fonts match

Hm, raises a point of whether we should change those `legend` default styling. Granted, not within the scope of this pull request.